Source/Core/VideoCommon/VertexLoaderManager.h

```diff
@@ -66,7 +69,31 @@ extern u32 g_current_components;
 extern BitSet8 g_main_vat_dirty;
 extern BitSet8 g_preprocess_vat_dirty;
 extern bool g_bases_dirty;  // Main only
-extern u8 g_current_vat;    // Main only
```

Comment:
I decided to check why this existed.

It was originally added as `last_id` for the software renderer in efbe5bc4b651d1f7f726bb064255e0e2562f7490, for `SetFormat`.

Later, it was also used to work around the debug cubes in Super Mario Sunshine: 51724c1ccdbc141b1aed67a2efc952e884cadd35 / d6ce8eef36f5b0bfd0078c51701b052cc9a77005. That hack was later reverted in c583cac5681409222c4557e48cbad8caedc17c4d in favor of GameINI stuff.

The parameters to `SetFormat` were removed in 3fc12431c5ed7a0df1e9bd4c538405a44f6561de (where I remarked that they hadn't been used since efbe5bc4b651d1f7f726bb064255e0e2562f7490, but I didn't realize that meant `last_id` was now unused).

The `g_current_vat` field was added in 1914087998bc296208355dc4b1b2553430b081b2 (likely due to a rebase mistake), and in d039b1bc0dfaba2a88036468b6d971bb1d7e463d (part of the same PR) I removed `last_id` in favor of it.

It's definitely useless now (and was useless for most of its existence), so removing it makes sense.